portfolio/src/main/webapp/script.js

```diff
@@ -8,7 +8,12 @@
  * @example loadSelector("#social", "socials.html")
  */
 function loadHTML(selector, filename) {
-  $(document).ready(function () {
+  $(document).ready(function() {
     $(selector).load(filename);
   });
 }
+async function getTopTenComment() {
```

Comment:
Are you calling getTopTenComment anywhere? 
I couldn't find any references to it. This could be the reason you don't see the comments on your page.

Reply:
I had added it as something that runs when the HTML load but that did not change anything, so I had taken it out.